pkg/operator/ceph/csi/spec.go

```diff
@@ -286,6 +290,15 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		tp.ForceCephFSKernelClient = "true"
 	}
 
+	// parese RPC timeout
+	timeout := k8sutil.GetValue(r.opConfig.Parameters, grpcTimeout, "150")
+	timeoutSeconds, err := strconv.Atoi(timeout)
+	if err != nil || timeoutSeconds < 120 {
```

Comment:
I think it's better to differentiate both conditions so we can do the correct logging message.

Reply:
Done.